modules/n1ql/pages/n1ql-language-reference/stringfun.adoc

```diff
@@ -1042,3 +1042,45 @@ SELECT UPPER("N1QL is awesome") as n1ql;
 }
 ----
 ====
+
+[#fn-str-upper]
+== URLENCODE(plain_string)
+
+=== Description
+Returns the input string encoded for use in a URL
+
+=== Arguments
+in_str:: A string, or any valid xref:n1ql-language-reference/index.adoc[expression] which evaluates to a string, that is the string to convert to upper case.
+
+=== Return Value
+A string representing the input string converted to upper case.
+
+=== Example
+====
+[source,text]
+----
+SELECT%20name%20FROM%20%60travel-sample%60.inventory.hotel%20LIMIT%201%3B
+----
+
+====
```

Comment:
Okay, you've shown the _result_ of the function here, but we need to see a complete example, with input and output.

```suggestion
====
[source,n1ql]
----
SELECT URLENCODE("SELECT name FROM `travel-sample`.inventory.hotel LIMIT 1;") AS encoded;
----

[source,json]
----
[
  {
    "encoded": SELECT%20name%20FROM%20%60travel-sample%60.inventory.hotel%20LIMIT%201%3B
  }
]
----
====
```